app/models/event_stream.rb

```diff
@@ -35,13 +35,21 @@ class EventStream < ApplicationRecord
 
   after_commit :emit_notifications, :on => :create
 
-  # TODO: Consider moving since this is EmsEvent specific. group, group_level and group_name exposed as a virtual columns for reports/api.
-  GROUP_LEVELS = %i(critical detail warning).freeze
```

Comment:
☝️ This was mixing EmsEvent group levels with EventStream.  Critical and warning were moved to EmsEvent, detail remains as a default here.